src/config/repository.php

```diff
@@ -51,4 +51,10 @@
      * use filters instead of null here.
      */
     'filter_key' => null,
+
+    /*
+     * This needs to be an array, valid options:
+     * repository, controller, model, transformer, policy, input, test
+     */
+    'default_stubs' => \Shamaseen\Repository\Commands\Generator::getDefaultFileGeneration(),
```

Comment:
> it would be better to put the getDefaultFileGeneration array directly in here as an array, so the user can easily pick/omit the ones he wants.

I use this in tests, to get the defaults and compare vs the selected, and check which files are not generated, we could keep that method, and repeat its contents here, which probably isn't hard to maintain, but can be error prone.